modules/virt-virtctl-commands.adoc

```diff
@@ -54,4 +60,10 @@ the specified port of the node.
 
 |`virtctl image-upload dv <datavolume_name> --size=<datavolume_size> --image-path=</path/to/image>`
 |Upload a virtual machine image to a new DataVolume.
+
+|`virtctl version`
+|Print the client and server version information.
+
+|`virtctl help`
+|Help on any command.
```

Comment:
I don't think "Help on any command" is really the most accurate description of what the command does, based on our Slack conversation, since it'd be better to use `-h` for help on a given command. This is also the only description that is a fragment. I think it should be changed to be parallel in structure with the others... maybe something like "Display a descriptive list of virtctl commands."

Reply:
yep. Changed to: "Display a descriptive list of `virtctl` commands."